src/Database/Driver.php

```diff
@@ -446,4 +446,23 @@ public function __debugInfo()
             'connected' => $this->_connection !== null
         ];
     }
+
+    /**
+     * Truncates an identifier if the database does
+     *
+     * @param string $key Aliased field name
+     *
+     * @return string Aliased field name as returned by Database
+     */
+    public function sanitizeIdentifier(string $key)
+    {
+        // In PostgreSQL, identifiers — table names, column names, constraint names, etc. — are limited to a
+        // maximum length of 63 bytes. Identifiers longer than 63 characters can be used, but they will be truncated
+        // to the allowed length of 63.
+        if ($this->_connection->getAttribute(PDO::ATTR_DRIVER_NAME) === 'pgsql') {
```

Comment:
Instead of a conditional in the base class, the base implementation should only return `$key` while the Postgres driver can apply this logic with no condition.

Reply:
sure, done!